openedx/core/djangoapps/schedules/tests/test_resolvers.py

```diff
@@ -172,47 +167,53 @@ def test_get_schedules_with_target_date_by_bin_and_orgs_filter_inactive_users(se
 
 
 @skip_unless_lms
-@skipUnless('openedx.core.djangoapps.schedules.apps.SchedulesConfig' in settings.INSTALLED_APPS,
-            "Can't test schedules if the app isn't installed")
 class TestCourseNextSectionUpdateResolver(SchedulesResolverTestMixin, ModuleStoreTestCase):
     """
     Tests the TestCourseNextSectionUpdateResolver.
     """
     def setUp(self):
-        super(TestCourseNextSectionUpdateResolver, self).setUp()
-        self.course = CourseFactory(highlights_enabled_for_messaging=True, self_paced=True)
-        self.yesterday = datetime.datetime.utcnow() - datetime.timedelta(days=1)
+        super().setUp()
         self.today = datetime.datetime.utcnow()
-        self.tomorrow = datetime.datetime.utcnow() + datetime.timedelta(days=1)
+        self.yesterday = self.today - datetime.timedelta(days=1)
+        self.course = CourseFactory(
+            highlights_enabled_for_messaging=True, self_paced=True,
+            start=self.today-datetime.timedelta(days=30)
+        )
 
         with self.store.bulk_operations(self.course.id):
-            ItemFactory.create(parent=self.course, category='chapter', highlights=[u'good stuff 1'], due=self.yesterday)
-            ItemFactory.create(parent=self.course, category='chapter', highlights=[u'good stuff 2'], due=self.today)
-            ItemFactory.create(parent=self.course, category='chapter', highlights=[u'good stuff 3'], due=self.tomorrow)
+            ItemFactory.create(parent=self.course, category='chapter', highlights=['good stuff 1'])
+            ItemFactory.create(parent=self.course, category='chapter', highlights=['good stuff 2'])
+            ItemFactory.create(parent=self.course, category='chapter', highlights=['good stuff 3'])
```

Comment:
We manually calculate due dates in our function anyway so these weren't doing anything